samples/package.json

```diff
@@ -2,10 +2,17 @@
   "name": "nodejs-scheduler-samples",
   "private": true,
   "main": "quickstart.js",
-  "files": ["*.js"],
+  "files": [
+    "*.js"
+  ],
   "license": "Apache-2.0",
+  "scripts": {
+    "test": "mocha system-test --timeout 10000"
+  },
   "dependencies": {
-    "@google-cloud/scheduler": "^0.1.2"
+    "@google-cloud/scheduler": "^0.1.2",
+    "body-parser": "^1.18.3",
```

Comment:
You most likely don't need body parser, as it's included in express by default in 4.16.0:
https://expressjs.com/en/changelog/4x.html

Reply:
done.